src/azul/indexer/__init__.py

```diff
@@ -52,3 +57,13 @@ def for_fqid(cls, fqid, manifest: MutableJSONs, metadata_files: MutableJSON) ->
     @property
     def fquid(self):
         return BundleFQID(self.uuid, self.version)
+
+    @abstractmethod
+    def drs_path(self, manifest_entry: JSON) -> Optional[str]:
```

Comment:
seems like you are anticipating https://github.com/DataBiosphere/azul/issues/2114

I'd rather not conflate that.

Also: If you change the signature here you need to touch every implementation